misc/systemd/ipfs.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+description=ipfs p2p daemon
+After=network.target
+Requires=network.target
+
+[Service]
+Type=simple
+User=ipfs
+RestartSec=1
+Restart=always
+PermissionsStartOnly=true
+Nice=18
+StateDirectory=/var/lib/ipfs
+Environment=IPFS_PATH=/var/lib/ipfs
+Environment=HOME=/var/lib/ipfs
+LimitNOFILE=8192
+Environment=IPFS_FD_MAX=8192
+EnvironmentFile=-/etc/sysconfig/ipfs
```

Comment:
The config file shouldn't exist at this path, see [here](https://www.freedesktop.org/software/systemd/man/file-hierarchy.html#System%20Packages)

Reply:
The dash means that this service file works fine even if the file doesn't exist.

Reply:
No, what I mean is `/etc/sysconfig/<program-name>` is not the convention for systemd service configuration.

Reply:
Shall I remove this line?  If not, to which value would you like it changed?

Reply:
sysconfig is a a redhat thing. The recommended way to customize a systemd unit (or set an environment variable) is to put a drop-in in `/etc/systemd/system/ipfs.service.d/`

Reply:
So, that line is actually sourcing the environment variables in `/etc/sysconfig/ipfs` (not related to the IPFS config).